README.md

```diff
@@ -34,18 +34,20 @@ python -m urdf2webots.importer --input=someRobot.urdf [--output=outputFile] [--n
 
 The script accepts the following arguments:
   - **-h, --help**: Show the help message and exit.
-  - **--input=INFILE**: Specifies the urdf file to convert.
+  - **--input=INPUT**: Specifies the URDF file to convert or the URDF content string.
```

Comment:
Instead of having a dual semantics for the `--input` parameter, it would be better to support only the file variant (not the the content string) and to read the content from `stdin` in case the `--input` command line is missing.